SPECS/freetds/freetds.spec

```diff
@@ -38,37 +36,34 @@ client to database server communications. FreeTDS includes call
 level interfaces for DB-Lib, CT-Lib, and ODBC.
 This package contains the libraries for %{name}.
 
-
 %package devel
-Summary: Header files and development libraries for %{name}
-Requires: %{name}-libs%{?_isa} = %{version}-%{release}
+Summary:        Header files and development libraries for %{name}
+Requires:       %{name}-libs%{?_isa} = %{version}-%{release}
 
 %description devel
 This package contains the header files and development libraries
 for %{name}. If you like to develop programs using %{name}, you will need
 to install %{name}-devel.
 
-
 %package doc
-Summary: Development documentation for %{name}
-BuildArch: noarch
+Summary:        Development documentation for %{name}
+BuildArch:      noarch
 
 %description doc
 This package contains the development documentation for %{name}.
 If you like to develop programs using %{name}, you will need to install
 %{name}-doc.
 
-
-%prep 
+%prep
 %setup -q
 
 #  correct perl path
-sed -i '1 s,#!.*/perl,#!%{__perl},' samples/*.pl
+sed -i '1 s,#!.*/perl,#!perl,' samples/*.pl
```

Comment:
Please revert this linter change. While it should work, we usually stick to absolute paths in shebangs.